### PR TITLE
addMultipleWalletCards

### PR DESCRIPTION
**Pull request by nimr77**

### CODE REVIEW

File: example/lib/data.dart [hunk 1]
```diff
@@ -0,0 +1,4 @@
+List<String> get data => [
+      "https://storage.cloud.google.com/ticketma-dev.appspot.com/passes/10911/6923491/pass_0.pkpass",
```
Comment (WebEferen) [inline example/lib/data.dart:2]: Access to this asset is forbidden.
Think about moving it into some other place (f.e drive google with full access to view).

Comment (nimr77) [reply example/lib/data.dart:2]: Sure, I didn't use it anyway I implemented it directly to the app

File: pubspec.yaml [hunk 2]
```diff
@@ -12,7 +12,10 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  pass_flutter: ^2.0.1
+  pass_flutter:
+    ^2.0.1
+    # pass_flutter:
+    # path: /Users/nimrsawafta/MyProjects/pass-flutter
```
[the remark this answers is not in the document]
Comment (WebEferen) [reply pubspec.yaml:18]: Avoid your personal details in here :)

**Comment by nimr77** [comment]
I have removed the un needed stuff

**Review by WebEferen — APPROVED** [review]

**Pull request merged**